README.md

```diff
@@ -3,17 +3,29 @@ Demo of media pipe python integration with Django
 
 ## Server Setup
 
-### GCP 
-- Temporary Server (Dwayne's Personal) `gcloud beta compute ssh --zone "us-east4-c" "mediapipe-server" --project "mystical-slate-241320"`
+### Docker / Local
+
+#### Mediapipe Container
+- Taken from https://google.github.io/mediapipe/getting_started/install.html
+- In a sibling folder, `git clone git@github.com:mitmedialab/mediapipe.git`
+- `docker build --tag=mediapipe .`
+- Quick Test (optiona1)
+  - `docker run -it --name mediapipe mediapipe:latest`
+  - `root@bca08b91ff63:/mediapipe# GLOG_logtostderr=1 bazel run --define MEDIAPIPE_DISABLE_GPU=1 mediapipe/examples/desktop/hello_world:hello_world`
+
+#### Demo Server Container
+- `docker-compose up`
+- `curl -d '{"archive_id": "013bc8ff-f069-44d9-af77-d170acd1c9c6"}' -H 'Content-Type: application/json' localhost:8080/hand`
+
+### Cloud Run
+- ...
+
+### Temporary GCP Server
```

Comment:
@Dwaynekj are you able to add the instructions for our group account yet or is that still being set up?

Reply:
Can't add them yet. I need Emily to add the credit card otherwise we can't use these resources